models/Element/Service.php

```diff
@@ -1263,6 +1263,21 @@ function (Concrete $object, Data $fieldDefinition, $property, $currentValue) {
         return $theCopy;
     }
 
+    /**
+     * @param mixed $properties
```

Comment:
Maybe nullable?
```suggestion
     * @param Model\Property[] $properties
```

Reply:
I thought about making it specific at first. Just wondering if this could be used for other types in future.

Reply:
Then we should at least use generics so that PhpStan/PhpStorm knows that the return value is the same as the parameter value.
```suggestion
     * @template T
     * @param T $properties
     * @return T
```